js/script.js

```diff
@@ -369,11 +375,27 @@ const renderTodos = () => {
 
     activateFilterButton(todos, filterState);
 
+    updateLocalStorageData();
```

Comment:
User applied search, filter and pagination and refreshed the screen. 
Will this logic persist all the state?

Reply:
No bhai, only filter state. So do I have to make all the state persistent?

Reply:
If it is I can implement thats too.

Reply:
Yes. Please